docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
       - HOSTNAME_EXTERNAL=localstack
 
   snowplow:
-    image: pocket/snowplow-micro:latest
```

Comment:
updated to prod image to run test against prod

Reply:
I wonder why that was done. Are Snowplow tests more reliable this way? This change will be detrimental for newer repositories (like Shareable Lists API) where the schemas don't exist on `prod` in Snowplow, only in `dev`/`latest`.

Reply:
Yeah I agree with Nina. We can switch it to `prod` when testing locally but should set it back to `latest`. I've merged this branch into my feature branch so I've made the change there.

Reply:
initially it was pointing to dev for prototyping this repo and some schema were wip in prod.

i feel the integration test should be pointing to `prod` schema as we care about correctness of the events able to publish for schema in prod.do you peeps see any reason why we schema would be delayed to be pushed in prod, after being pushed to dev? can we work with analytics team to publish them to prod? 

the circleCI still runs against prod (as it should), so if the test passes here locally, the circleCI will still fail.